tests/cypress/e2e/for-teacher_page/class_page/create_accounts_page/remove_row.cy.js

```diff
@@ -0,0 +1,24 @@
+import {loginForTeacher} from '../../../tools/login/login.js'
+import {createClass} from '../../../tools/classes/class.js'
+
+
+describe('Is able to go to logs page', () => {
+  it('Passes', () => {
+    loginForTeacher();
+    cy.wait(500);
+    createClass();
+    cy.get(".view_class").first().click(); // Press view class button
+    cy.get('#add-student').click();
+    cy.get('#create-accounts').click(); 
+    cy.get(':nth-child(2) > #username').type("student10");
+    cy.get(':nth-child(2) > #password').type("123456");
+    cy.wait(1000);
+    cy.get(':nth-child(2) > #username').should('have.value', 'student10');
+    cy.get(':nth-child(2) > .fill-current > path').click();
```

Comment:
What does this do? Could you add a comment or use less generic classes to call the element?